### PR TITLE
Added functionality for the add command, all contained within the Wri…

### PR DESCRIPTION
**Pull request by hgermundsen**
…teList func

### CODE REVIEW

File: bot/bot.go [hunk 1]
```diff
@@ -66,9 +66,14 @@ func (b *Bot) ProcessMessage(w http.ResponseWriter, r *http.Request, ps httprout
...
 			b.SendMessage(fullList)
+		} else if input[:3] == "add" {
+			//TODO: Make this a lot better to handle big brain users
+			list.WriteList([]string {input[4:]})
+		
 		} else {
 			b.SendMessage(fmt.Sprintf("Repeating what you said: %s", groupmeMessage.Text))
 		}
 
+
```
Comment (nchaloult) [inline bot/bot.go:77]: Can we get rid of this extra line?

File: list/list.go [hunk 2]
```diff
@@ -23,3 +24,20 @@ func ReadList() []string {
...
+	if err != nil {
+		//TODO: Fix this error handling
+		panic(err)
+	}
+	err = ioutil.WriteFile("list.json", jsonAsBytes, 0644)
+	if err != nil {
+		//TODO: Fix this error handling
+		panic(err)
+	}
+
```
Comment (nchaloult) [inline list/list.go:42]: This line too

File: bot/bot.go (hunk 1 above)
Comment (hgermundsen) [reply bot/bot.go:77]: Fixed that one

File: list/list.go (hunk 2 above)
Comment (hgermundsen) [reply list/list.go:42]: Fixed this one too

**Comment by nchaloult** [comment]
👍

**Pull request merged**